MinimalCirclesPath.py

```diff
@@ -64,33 +64,33 @@ def getDistance(startLat, startLong, destLat, destLong):
 
 # pathWidth is the desired scaned distance from play diameter, radius is radius of circle passed to NOTAM
 def getPath( startLat, startLong, destLat, destLong, radius, pathWidth):
-
-    # TODO For actual application you would need to capture N/S/E/W
-
     
-      
-    # gets total distance from start to finish
-    totalDistance = getDistance(startLat, startLong, destLat, destLong)
-
     # gets step distance (pythagorean theorum)
     stepDistance = 2 * sqrt((radius)**2-(pathWidth/2)**2)
 
     # gets direction
     bearing = calculateBearing(startLat, startLong, destLat, destLong)
 
+    # updates the start so the area in the opposite direction of the path is right on the edge of the first circle
+    updatedStart = nextPoint(startLat,startLong, bearing, radius - (pathWidth/2) )
+
+    # updates the dest so that it is the furthest point on the path. This means going the pathWidth/2 past the dest
+    updatedDest = nextPoint(destLat, destLong, bearing, (pathWidth/2))
 
-    # gets point to call NOTAM and include all of starting area from farther down path
-    startPoint = nextPoint(startLat,startLong, bearing, radius - (pathWidth/2) )
+    # gets total distance from start to finish
+    totalDistance = getDistance(updatedStart[0], updatedStart[1], updatedDest[0], updatedDest[1])
 
     # List to return with path of points from start to dest
-    coordList = [startPoint]
+    coordList = [updatedStart]
 
-    # loops for each step until passed the destination
-    for _ in range(floor((totalDistance - (radius - (pathWidth/2)))/stepDistance)):
+    # loops for each step until passed the destination. range(split total distance minus area covered from start point)
+    for _ in range(ceil((totalDistance - (stepDistance/2))/stepDistance)):
         nextCircle = nextPoint(coordList[-1][0],coordList[-1][1], bearing, stepDistance)
```

Comment:
I have a code-readability problem with a function called "nextPoint" which returns a tuple of lat/long coordinates which you assign to a variable named ... of all things ... `nextCircle`.  Circles are not Points, Points are not Circles.  Even if Finkle is Einhorn.

I actually think the code would read much better by using Python's super-helpful tuple unpacking behavior:
```python
nextLat, nextLong = nextPoint(coordList[-1][0], coordList[-1][1], bearing, stepDistance)
coordList.append( (nextLat,nextLong) )
bearing = calculateBearing(nextLat, nextLong, destLat, destLong)
```